src/service/resource.rs

```diff
@@ -93,6 +93,7 @@ pub enum ResourceType {
     MatterFabric,
     Motion,
     PublicImage,
+    PrivateGroup,
```

Comment:
Reference for this?

Reply:
All I can provide is the JSON returned by my bridge which contains this resource type (in the owner's rtype)
![image](https://github.com/user-attachments/assets/b2df3023-b69f-4248-a73d-813c06fa7808)
I used some experiment-level features in the past; it may be residue from it.

Reply:
Gotcha. If you wouldn't mind, could you add this API response to the [test fixtures directory](https://github.com/rektdeckard/hues/tree/main/test/fixtures/resource)? There are a lot of things I simply can't test on my own, either because I don't have the hardware, or simply haven't had the need to configure my own bridge that way yet. My hope is to make this lib a little more confident by testing against different people's real-world configurations.

Reply:
At the moment, we only test that we can parse various responses from `/clip/v2/resource`, but will add others!

Reply:
Yes, I will add it.